src/locales/zh-TW.json

```diff
@@ -105,10 +105,10 @@
     "clearCanvasMessage": "如果重載還是無法解決，嘗試 ",
     "clearCanvasMessage_button": "清理畫布",
     "clearCanvasCaveat": " 這會導致目前的進度遺失 ",
-    "openIssueMessage_pre": "Before doing so, we'd appreciate if you opened an issue on our ",
-    "openIssueMessage_button": "bug tracker.",
-    "openIssueMessage_post": " Please include the following error stack trace (and if it's not private, also the scene content):",
-    "errorStack": "Error stack trace:",
+    "openIssueMessage_pre": "",
```

Comment:
This is great! It's making it more obvious what's missing

Reply:
yeap! there was this one setting :)